search-setup/package.json

```diff
@@ -1,10 +1,13 @@
 {
   "type": "module",
   "scripts": {
-    "dev": "tsc && node --env-file=.env main.js"
+    "create-index": "tsc && node --env-file=.env main.js"
   },
   "devDependencies": {
+    "@types/jest": "^29.5.14",
     "@types/node": "^22.10.2",
+    "jest": "^29.7.0",
+    "ts-jest": "^29.2.5",
```

Comment:
Why install jest-related dependencies? There's nothing that runs using Jest, is there?

Reply:
I made the changes on the unit testing branch, then merged it into this one, so this must've pulled in as well. Leaving as is because the other branch will use the same dependencies when this gets merged

Reply:
Fair enough, I suspected this might be the case after I made the comment!